src/types/types.tsx

```diff
@@ -1,27 +1,27 @@
-import firebase from "firebase/app";
+import firebase from 'firebase/app';
 
 export type User = {
-    id: string;
-    access: string;
-    createdJobs: Job[];
-    email: string;
-    likedJobs: Job[];
-    name: string;
-    phoneNumber: string;
-    verified: boolean;
-    // note: is password necessary here or in firebase?
-    password: string;
-}
+  id: string;
+  access: string;
+  createdJobs: string[]; //switched to string of jobIds to match Firebase
+  email: string | null;
+  likedJobs: string[]; //switched to string of jobIds to match Firebase
+  name: string;
+  phoneNumber: string | null;
+  verified: boolean;
+  // note: is password necessary here or in firebase?
+  password: string | null;
```

Comment:
ideally this shouldn't be here — we should tie this object to Firebase Auth User objects through the UID

Reply:
done